enclave-tls/src/enclave_quotes/sgx-ecdsa-qve/verify_evidence.c

```diff
@@ -0,0 +1,163 @@
+/* Copyright (c) 2020-2021 Alibaba Cloud and Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <enclave-tls/log.h>
+#include <enclave-tls/enclave_quote.h>
+#include <sgx_urts.h>
+#include <sgx_quote.h>
+#include <sgx_quote_3.h>
+#include <sgx_ql_quote.h>
+#include <sgx_dcap_quoteverify.h>
+#include "sgx_ecdsa.h"
+#include "sgx_stub_u.h"
+
+void getRandomNonce(uint8_t *nonce, uint32_t size)
+{
+    for(unsigned int i = 0; i < size; i ++)
+        nonce[i] = (uint8_t)((rand() % 255) + 1);
+}
+
+enclave_quote_err_t sgx_ecdsa_qve_verify_evidence(enclave_quote_ctx_t *ctx,
+					      attestation_evidence_t *evidence,
+					      uint8_t *hash, unsigned int hash_len)
```

Comment:
I think there are still many code chunks which can be shared with sgx-ecdsa. Please split the codes of sgx-ecdsa into chunks and re-uses them in sgx-ecdsa-qve/verfiy_evidence.c.